### ISSUE TITLE
Task never defined: default, watch

### ISSUE THREAD

**Issue by bonovski** [gulpjs/gulp-cli#191]
Updated to gulp-cli 2.2.0 today and now my default and watch tasks are not recognised anymore. Isn't this syntax valid anymore?

```
// Default task
exports.default = series( clean, parallel (styles, scripts, images)  );

// Watch task
exports.watch = series( clean, parallel (styles, scripts, images), serve, watch );
```

**Comment by phated** [comment]
They should be fine, but you have to upgrade your local undertaker  (which might not be updated due to a lockfile). Cc @coreyfarrell who worked on those changes.

I think I might need to cut a gulp release to ensure people have the new undertaker.

**Comment by bonovski** [comment]
When I list my tasks I get this:
```
[16:53:00] Tasks for ~/Sites/staging/wp-content/themes/theme/gulpfile.js
[16:53:00] └─┬ <series>
[16:53:00]   └─┬ <series>
[16:53:00]     ├── clean
[16:53:00]     └─┬ <parallel>
[16:53:00]       ├── styles
[16:53:00]       ├── scripts
[16:53:00]       ├── images
```
Instead of `default` it prints out `<series>` twice.

After updating `undertaker` locally and manually everything's working fine again.
You definitely should bump `gulp` up a notch to get the new `undertaker` automatically.
Thanks for the quick help.

**Comment by phated** [comment]
I believe lockfiles will still cause issues but I've been meaning to cut a release anyway. I'll do that when I get back to my computer.

**Comment by phated** [comment]
I just released gulp 4.0.1 - thanks for letting us know about the issue.
